test/recipes/hooks/run.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+export KITCHEN_HOOK=$1
+export KITCHEN_ROOT_DIR=$2
+
+echo "*** Run ${KITCHEN_HOOK} for suite ${KITCHEN_SUITE_NAME} on host ${KITCHEN_INSTANCE_HOSTNAME}"
```

Comment:
Question: where is it defined `KITCHEN_INSTANCE_HOSTNAME` ? Is it set in the environment by kitchen?

Reply:
Yes, same as above.